download/conftest.py

```diff
@@ -0,0 +1,5 @@
+import pytest
+
+pytest_plugins = [
```

Comment:
I'm not familiar with using fixtures in pytest, so it might be useful to add some comments to give a general idea of what this is for.